models/DataObject/AbstractObject.php

```diff
@@ -406,9 +406,9 @@ public function getSiblings(
                 if ($this->getId()) {
                     $list->addConditionParam('id != ?', $this->getId());
                 }
-                $list->setOrderKey('key');
                 $list->setObjectTypes($objectTypes);
```

Comment:
Also documents https://github.com/pimcore/pimcore/blob/551efacfc2323a93a10cca803bf354c886c12509/models/Document.php#L595-L596
and assets https://github.com/pimcore/pimcore/blob/551efacfc2323a93a10cca803bf354c886c12509/models/Asset.php#L869-L870
seem needing the same change

Reply:
Mhhh, didn't know that these `getChildrenSortBy` were only featured in objects.

The dilemma now is, `getSiblings` is consistent with `getChildren`, but these 2 are not consistent with the Document and Assets anymore 🤔